app/models/rdv.rb

```diff
@@ -289,6 +289,15 @@ def synthesized_receipts_result
 
   delegate :domain, to: :organisation
 
+  def soft_delete
+    self.skip_webhooks = true
```

Comment:
Un petit commentaire permettrait de clarifier cette ligne qui peut sembler contre-intuitive.

```suggestion
    # disable the :updated webhook because we want to manually trigger a :destroyed webhook
    self.skip_webhooks = true
```